src/libs/Network.js

```diff
@@ -78,7 +131,10 @@ function processNetworkRequestQueue() {
 
         // Check to see if the queue has paused again. It's possible that a call to enhanceParameters()
         // has paused the queue and if this is the case we must return.
-        if (shouldSkipRequest) {
+        if (!shouldMakeRequest(queuedRequest)) {
```

Comment:
I guess not. Mostly kept it the same so we would get the logs in `shouldMakeRequest()` in here. But maybe those should be inline and tell us at what point we are skipping.

Reply:
But realizing that is happening in `shouldRetryRequest` anyway now. Yep will fix this.

Reply:
Actually, I think we are not supposed to retry these on second thought since the scenario we are trying to cover is this

https://github.com/Expensify/ReactNativeChat/blob/4d15b34115bae14fe14fa46aa3e93ea9ff8726c3/src/libs/API.js#L53-L60

and we are logging the user out in that case so we actually do want to toss these requests.

Reply:
Actually removing this broke everything. Good times.

Reply:
Ok I remember why now. heh. _Some_ requests are allowed to be made when the network queue is paused e.g. if they have `forceNetworkRequest === true`. So even though we already checked this we can't just check for `isQueuePaused` or else the requests that should run will never run.